src/components/Modals/DeleteContactModal.jsx

```diff
@@ -77,12 +59,7 @@ const DeleteContactModal = ({
       aria-describedby="dialog-description"
```

Comment:
The `aria-labelledby` is not linked to anything again. Will be when use `<DialogTitle />` instead of `<FormSection />`         Can probably switch it now if not using the spinner from `FormSection`.